tests/test_validation_interface/test_repo_tokens_mutually_exclusive_work.py

```diff
@@ -0,0 +1,33 @@
+import os
+import pytest
+
+from .utils import initialize_repo
+from fiasko_bro import CodeValidator
+
+
+@pytest.fixture(scope='module')
+def code_validator():
+    code_validator = CodeValidator()
+    return code_validator
+
+
+@pytest.fixture(scope='module')
+def origin_repo():
+    repo_path = 'test_fixtures{}general_repo_origin'.format(os.path.sep)
+    initialize_repo(repo_path)
+    return repo_path
+
+
+def test_mark_repo_with_both_fail(origin_repo, code_validator):
```

Comment:
It's better to place these tests in `test_tokenized_validators_work.py`. And we only need `test_mark_repo_with_both_fail`, two others cases are implicitly tested in `test_tokenized_validators_work.py`.

Tests in `test_validation_interface` are testing a big chunk of the system. They do not need to be as granular as unit tests. This is something I should've documented.

More on the difference between different types of tests: https://stackoverflow.com/a/4904533/3694363.